LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -191,10 +197,12 @@ module LibraBFT.ImplShared.Consensus.Types where
 
   -- TODO-1? We would need lenses to be dependent to make a lens from round
   -- managers to block stores.
-
   rmGetBlockStore : (rm : RoundManager) → BlockStore (α-EC-RM rm)
   rmGetBlockStore rm = (₋rmWithEC rm) ^∙ (epBlockStore (α-EC-RM rm))
 
+  rmSetBlockStore : (rm : RoundManager) → BlockStore (α-EC-RM rm) → RoundManager
```

Comment:
This does not appear to be used.
Let's delete it if not needed.

Reply:
I make use of this in a branch that extends this one, and it will be needed to implement `BlockStore.executeAndInsertBlockM` (since we cannot write a lens for the block store). To leave it in that branch though would might lead to further progress on developing the implementation model adding it independently, leading to an unnecessary (though minor) merge conflict.

Reply:
If you plan on using it then no need to remove.